config/rbac/dedicated-admin-role.yaml

```diff
@@ -0,0 +1,19 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: df-dedicated-admin-role
+  labels:
+    managed.openshift.io/aggregate-to-dedicated-admins: cluster
```

Comment:
```suggestion
	# The suffix "dedicated-admins" is the cluster role that the rules below will get appended to.
	# This is specified and validated here: https://github.com/openshift/managed-cluster-config/blob/f4537242f9c54ee81bd049c458f3dc1e7ed9f838/deploy/rbac-permissions-operator-config/03-dedicated-admins-cluster.ClusterRole.yaml#L5-L8
    managed.openshift.io/aggregate-to-dedicated-admins: cluster
```

Reply:
@rexagod we are not using the suffix here it the name in metadata is used for rule binding, I will add a proper comment

Reply:
The `aggregate-to-[cluster role]` is used for role aggregation, no? I inferred that from the text below. Just for my own curiosity, PLMK if that's not the case, thanks.

![image](https://user-images.githubusercontent.com/33557095/176110744-e40d0af7-19b4-4517-a309-315e42b88fdb.png)